apps/versions/urls.py

```diff
@@ -6,8 +6,10 @@
 
 urlpatterns = patterns(
     '',
-    url('^$', views.version_list, name='addons.versions'),
-    url('^format:rss$', VersionsRss(), name='addons.versions.rss'),
+    url('^(?:status:(?P<status>[^/]+))?$',
```

Comment:
oh, didn't realize the url you wanted it to be was `/versions/status:beta` and not simply `/version/beta`...

How about `/versions` and `/beta-versions`?